modules/ROOT/pages/index-migration.adoc

```diff
@@ -5,25 +5,16 @@ endif::[]
 
 // author: Dan D
 
-Mule 4 improvements make it easier to learn, develop, and manage than Mule 3. For an overview of these capabilities, you should read xref:mule-runtime-updates.adoc[What's New in Mule 4].
+Mule 4 improvements make it easier to learn, develop, and manage than Mule 3.
+Benefits include:
 
-This documentation describes the differences between Mule 3 and Mule 4, indicates when you should migrate to Mule 4, and shows what it takes to migrate apps manually. It includes information on these topics:
+* Easier and faster app development
+* Seamless access and data mapping
+* Simplified connectivity
+* Auto-tuned and highly performant runtime
+* Seamless upgrades of runtime and connector versions
```

Comment:
runtime > any of the accepted forms: Mule, Mule runtime engine

Reply:
done via suggestion

Reply:
done via suggestion